_redirects

```diff
@@ -1,3 +1,6 @@
+
+/docs/build/hooks-operations  /docs/build/hooks-operations 301
```

Comment:
Are these supposed to be the same path?

Reply:
oh boy, i think i was being overly cautious here. I'll remove